content/maintainers-guide/project-organization.md

```diff
@@ -34,6 +34,12 @@ Current maintainers:
 - Thomas Baccelli ([@atomiix](https://github.com/atomiix))
 - Valentin Szczupak ([@NeOMakinG](https://github.com/NeOMakinG))
 
+#### Scoped Maintainers
+
+Scoped Maintainers have the same mission as Project Maintainers, but over a limited scope (e.g. a project, a repository, or a technology).
```

Comment:
what does that mean exactly? Specific scope maintainers will have access to specific repository? A scope  maintainer (Ex.: user doc)  can merge a PR on the Core repository?

Reply:
Yes

> have the same mission as Project Maintainers
> but over a limited scope

:-)

Reply:
@kpodemski  limited scope does not necessarily mean limited rights, so it can introduce ambiguity if it is not clear.

Reply:
Let's improve it to clarify that a scope is a repository :)

Reply:
I created an item for this https://github.com/PrestaShop/open-source/issues/89